vendors/circom/circomlib/wtns/wtns_unittest.cc

```diff
@@ -31,15 +31,14 @@ TEST(WtnsTest, Parse) {
   };
   EXPECT_EQ(wtns->ToV2()->header, expected_header);
 
-  v2::WtnsDataSection<F> expected_data = {std::vector<F>{
-      F(1),
-      F(60),
-      F(3),
-      F(4),
-      F(5),
-      F(12),
-  }};
-  EXPECT_EQ(wtns->ToV2()->data, expected_data);
+  std::vector<F> expected_data{
+      F::FromMontgomery(F(1).ToBigInt()), F::FromMontgomery(F(60).ToBigInt()),
+      F::FromMontgomery(F(3).ToBigInt()), F::FromMontgomery(F(4).ToBigInt()),
+      F::FromMontgomery(F(5).ToBigInt()), F::FromMontgomery(F(12).ToBigInt()),
+  };
```

Comment:
```suggestion
  std::vector<F> expected_data{
      F(1), F(60), F(3), F(4), F(5), F(12),
  };
```

Reply:
Witnesses are stored in unreduced montgomery form, this transformation is needed!